saskatoon/sitebase/templates/app/detail_views/harvest/status.html

```diff
@@ -10,11 +10,46 @@
                 <div class="wb-traffic-inner notika-shadow sm-res-mg-t-30 tb-res-mg-t-30">
                     <div class="website-traffic-ctn">
                         <p>{% translate 'Harvest status' %}</p>
+                        {% if harvest.pick_leader is None or harvest.pick_leader.id != request.user.id %}
                         <h2>{{ harvest.status }}</h2>
+                        {% else %}
+                        <div class="dropdown">
+                            <button
+                            class="btn dropdown-toggle"
+                            type="button"
+                            id="harvest-status-dropdown"
+                            data-toggle="dropdown"
+                            >
+                            <h2>
+                                {{ harvest.status }}
+                                <span class="caret"></span>
+                            </h2>
+                            </button>
+                            <ul
+                            class="dropdown-menu"
+                            role="menu"
+                            aria-labelledby="menu1"
+                            >
+                                {% for status_option in status_options %}
+                                <li role="presentation">
+                                    <a
+                                    href="{% url 'harvest-status-change' harvest.id %}?status={{ status_option }}"
+                                    role="menuitem"
+                                    tabindex="-1"
+                                    >
+                                    {{ status_option }}
+                                </a>
+                                </li>
```

Comment:
I get a bottom padding that is a bit too large on Firefox
![screenshot_2022-07-21_17:40:15](https://user-images.githubusercontent.com/41802320/180319116-8513deb0-cf79-4b45-aceb-d0d4fb9fd048.png)

Also would it be easy to change the color of the dropdown background based on the status to match the colors used in the calendar? I think it could be really useful.

Reply:
I was thinking about that too. Great idea!

Reply:
@2ynn What do you think about this?

![image](https://user-images.githubusercontent.com/52796958/180648942-9e3a5bd1-4bfb-4d4d-8a50-6caf9e4d02d6.png)
![image](https://user-images.githubusercontent.com/52796958/180648961-ef50f74c-bc4e-4496-bcea-861b7ab18a5f.png)
![image](https://user-images.githubusercontent.com/52796958/180648983-456cb9ee-7e1f-411e-a34f-e61b1275aa53.png)

**The rest of the statuses**:
![image](https://user-images.githubusercontent.com/52796958/180648995-d3f84224-f60e-4999-9c1a-e1e1ad1af32e.png)